packages/mdc-drawer/package-lock.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
I'm curious, did this file become a thing just from a top-level npm install, or did an npm install get performed within packages/mdc-drawer? I've seen this pop up a couple of times in specific packages but most of our packages don't have them, and we've gotten in the habit of removing them from inner folders (I've seen git tend to trip over them when switching between feature branches).